### PR TITLE
refactor: unnecessary live post notification

### PR DESCRIPTION
**Pull request by sshanzel**
**Blocked**: until [preview PR in FE](https://github.com/dailydotdev/apps/pull/1744) is merged.

Sync external link submission post-release removal of the post_live notification.

### CODE REVIEW

**Review by rebelchris — CHANGES_REQUESTED** [review]
This looks good.
Just a suggestion for full clean up

File: src/workers/notifications/postAdded.ts [hunk 1]
```diff
@@ -73,19 +66,6 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
-
-        if (post.type === PostType.Share) {
-          // squad_post_live notification
-          if (baseCtx.sharedPost?.origin === PostOrigin.Squad) {
-            notifs.push({
-              type: 'squad_post_live',
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply src/workers/notifications/postAdded.ts:81]: I tried searching the codebase but I don't seem to see any additional trigger points. Do you remember the specific place/use case?

[the remark this answers is not in the document]
Comment (rebelchris) [reply src/workers/notifications/postAdded.ts:81]: Its not trigger, mainly the actual notifications related to it.

[the remark this answers is not in the document]
Comment (sshanzel) [reply src/workers/notifications/postAdded.ts:81]: Ah, you mean removing it on the notifications generator? Thought of it too but refrained from doing so, but if we don't see any use for it in the future, then I'd be glad to clean those up 🚀

[the remark this answers is not in the document]
Comment (rebelchris) [reply src/workers/notifications/postAdded.ts:81]: Personally don't see any use as it won't ever be triggered.
We can always reintroduce from the history if need be.

**Review by rebelchris — APPROVED** [review]
Nice work :)

**Review by idoshamun — APPROVED** [review]

**Pull request merged**